src/sdam/topology.ts

```diff
@@ -206,38 +190,24 @@ export class Topology extends EventEmitter {
   static readonly CONNECT = 'connect' as const;
 
   /**
-   * @param seedlist - a string list, or array of ServerAddress instances to connect to
+   * @param seedlist - a list of HostAddress instances to connect to
    */
-  constructor(seedlist: string | ServerAddress[], options?: TopologyOptions) {
+  constructor(seedlist: string | HostAddress | HostAddress[], options: TopologyOptions) {
```

Comment:
Okidoke, I fixed this, our tests can get away with specifying no options but the defaulting work in the driver should always be created by the all this options processing, so I left the typings to still require a second argument. In the future when we type our tests this will cause an issue but I feel like at the time is when we can repair the config.newTopology function to do the defaulting there and stop calling the topology constructor directly.